src/cephadm/cephadm

```diff
@@ -1270,10 +1272,26 @@ def extract_uid_gid(img='', file_path='/var/lib/ceph'):
     (uid, gid) = out.split(' ')
     return (int(uid), int(gid))
 
+def deploy_extra_files(extra, data_dir, uid, gid):
```

Comment:
as we now use the mon store to keep the cert and key, do we still need to deploy extra files at all right now?

Reply:
yeah, we might be able to loose this. Still trying to get the thing running. ATM when I run this in my vstart env, the RGW in the container fails to talk to monitor config. 

Or rather, when I jump into the docker container and pass correct beast details it fails:

```
# /bin/radosgw -d -setuser ceph --setgroup ceph --rgw-region=default --rgw-zone=default --keyring=/var/lib/ceph/radosgw/ceph-rgw.default.default.ironic-moliver.rslfos/keyring --rgw_frontends=beast port=8888 ssl_port
=4443 ssl_certificate=config://rgw/cert/default/default.crt ssl_key=config://rgw/cert/default/default.crt -d                 
2020-03-12T00:30:59.367+0000 7ff527191700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
2020-03-12T00:30:59.367+0000 7ff526990700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
2020-03-12T00:30:59.367+0000 7ff52618f700 -1 monclient(hunting): handle_auth_bad_method server allowed_methods [2] but i only support [2,1]
failed to fetch mon config (--no-mon-config to skip)    
```

Reply:
did you pass the path to the vstart ceph.conf?